app/views/homepages/index.html.erb

```diff
@@ -0,0 +1,49 @@
+<div class='homepage'>
+  <section class="spotlight">
+  <h2>
+  <span class="spotlight__header--prefix">Media Spotlight</span>
+  <%=link_to @spotlight.title%> by <%=@spotlight.creator%>
+  </h2>
+  <p><%=@spotlight.votes.count%> votes - <%=@spotlight.description%></p>
+  </section>
+  <section class="top-ten__container">
+    <section class="top-ten__list-container">
+      <h2 class="top-ten__header">Top Movies</h2>
+        <ul class="list-group top-ten__list">
+          <%@top_ten_movie.each do |movie|%>
+            <li class="list-group-item">
+            <h4><%=link_to movie.title, work_path(movie.id)%>
+            <small class="top-ten__creator"> by: <%=movie.creator%></small>
+            </h4> 
+            <p><%=movie.votes.count%> votes</p>
+            </li>
+          <%end%>
+        </ul>
+    </section>
+    <section class="top-ten__list-container">
+      <h2 class="top-ten__header">Top Books</h2>
+        <ul class="list-group top-ten__list">
+          <%@top_ten_book.each do |book|%>
+            <li class="list-group-item">
+            <h4><%=link_to book.title, work_path(book.id)%>
+            <small class="top-ten__creator">by: <%=book.creator%></small>
+            </h4> 
+            <p><%=book.votes.count%> votes</p>
+            </li>
```

Comment:
This is a great place to use a view partial, as you have the same html repeated 3 times!!!